instrumentation/redisson/redisson-common/testing/build.gradle.kts

```diff
@@ -10,5 +10,5 @@ dependencies {
   implementation("org.spockframework:spock-core")
   implementation("org.testcontainers:testcontainers")
 
-  compileOnly("org.redisson:redisson:3.0.0")
+  compileOnly("org.redisson:redisson:3.7.2")
```

Comment:
is this needed?

Reply:
It is deliberate. New test uses classes that are not present in 3.0.0. Tests are still run agains 3.0.0 there is a Class.forName check to skip test when class is not present during runtime.

Reply:
oh, right, this is in `testing` artifact, thx 👍